style.css

```
@@ -0,0 +1,58 @@
+*{
+    margin: 0;
+    bottom: 0;
+    box-sizing: border-box;
+}
+
+
+body{
+    background-color: rgb(84, 38, 127);
+    text-align: center;
+    font-family: 'Roboto', sans-serif;
+        
+}
+
+h1{
+    
+    font-size: 25px;
+    color: rgb(219, 214, 228);
+    margin-top: 25px;
+  
+}
+
+#screen{
+border: 5px solid rgb(61, 3, 92);
+width: 350px;
+margin: 80px auto;
+background-color: rgb(15, 7, 25);
+opacity: 0.8;
+height: 450px;
+border-radius: 8px;
+
+
+}
+input, .buttom{
```

Let's avoid using tags directly. replace the `input` selector with a meaningful class.

---

also you have a typo 
```suggestion
input, .button{
```